osutil/mountinfo_linux_test.go

```diff
@@ -131,6 +131,16 @@ func (s *mountinfoSuite) TestParseMountInfoEntry4(c *C) {
 	c.Assert(err, ErrorMatches, `cannot parse device major:minor number pair: "corrupt"`)
 }
 
+// Check that \r is parsed correctly.
+func (s *mountinfoSuite) TestParseMountInfoEntry5(c *C) {
+	real := "2074 27 0:54 / /tmp/strange\rdir rw,relatime shared:1039 - tmpfs tmpfs rw"
```

Comment:
Should we add a test with octal escaped sequence too?

Reply:
No, we should not. We have those already :)
The Go side was always escaping those, the C side didn't because it doesn't really matter for us.

https://github.com/snapcore/snapd/blob/master/osutil/mountinfo_linux_test.go#L86

Reply:
Ah, missed that. Thanks!